scripts/generate_openapi.py

```
@@ -0,0 +1,621 @@
+#!/usr/bin/env python3
+"""
+Python script to generate OpenAPI JSON from evaluators.generated.ts
+"""
+
+import json
+import re
+import sys
+from pathlib import Path
+from typing import Dict, Any, List, Optional
```

Ruff (0.12.2)

`typing.List` imported but unused

Remove unused import

(F401)